app/bundles/LeadBundle/Entity/PointsChangeLog.php

```diff
@@ -87,10 +87,8 @@ public static function loadMetadata(ORM\ClassMetadata $metadata): void
 
     /**
      * Get id.
-     *
-     * @return int
      */
-    public function getId()
+    public function getId(): int
     {
         return (int) $this->id;
     }
```

Comment:
Here, it's easier to see

Reply:
I know. I was wondering whether we should instead remove the typecasting to int. It should be a string. But then it could snowball to other changes like changing some old unit tests and perhaps PHPSTAN would found other issues. Maybe ignoring all entities would be the safest bet for now.

Reply:
Removing the typecasting would be changing behavior now.
I'd like to avoid that as it's a different to upgrade as it's a different process with different rules.

If you want to change the entity behavior, I'd suggest doing that in a different PR to make it easier debug later. Personally I want to avoid any refactoring at all as very risky without being the project owner.